src/current/v23.2/work-with-virtual-clusters.md

```diff
@@ -0,0 +1,197 @@
+{{site.data.alerts.end}}
+
+### DB Console
+
+This section shows how to connect using DB Console when cluster virtualization is enabled.
+
+{{site.data.alerts.callout_success}}
+If the same SQL user has the `admin` role on the system interface and also has roles in the virtual cluster, that user can switch between them from the top of the DB Console.
+{{site.data.alerts.end}}
+
+#### Connect to a virtual cluster
+
+By default when you connect using the DB Console, you are logged into the `application` virtual cluster. You can optionally add the `GET` URL parameter `optionsl=-ccluster=application` to the DB Console URL so that the intention to connect to the `application` virtual cluster is more clear.
+
+#### Connect to the system interface
+
+To connect to the system interface using the DB Console, add the `GET` URL parameter `optionsl=-ccluster=system` to the DB Console URL.
+
+### SQL clients
+
+This section shows how to connect using `cockroach sql` when cluster virtualization is enabled.
+
+#### Connect to a virtual cluster
+
+By default when you connect using a SQL client, you are logged into the `application` virtual cluster. You can optionally add the `optionsl=-ccluster=application` option to the connection string so that the intention to connect to the `application` virtual cluster is more clear. Replace `virtual_cluster_name` with the name of the virtual cluster.
+
+
+For example, to connect to the `application` virtual cluster using the `cockroach sql` command:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach sql --url \
+"postgresql://root@{node IP or hostname}:26257/?options=-ccluster=application&sslmode=verify-full" \
+--certs-dir "certs"
+~~~
+
+#### Connect to the system interface
+
+{{site.data.alerts.callout_info}}
+You should only connect to the system interface for cluster administration. To work with databases, tables, or workloads, connect to the application virtual cluster.
+{{site.data.alerts.end}}
+
+To connect to the system interface, pass the `options=-ccluster=system` parameter in the URL. You must have the `admin` role on the system interface.
+
+For example, to connect to the system interface using the `cockroach sql` command:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach sql --url \
+"postgresql://root@{node IP or hostname}:26257/?options=-ccluster=system&sslmode=verify-full" \
+--certs-dir "certs"
+~~~
+
+#### Grant access to the system interface
+
+To grant access to the system interface, you must connect to the system interface as a user with the `admin` role, then grant the `admin` role to the SQL user.
+
+{{site.data.alerts.callout_info}}
+To prevent unauthorized access, you should limit the users with access to the system interface.
+{{site.data.alerts.end}}
+
+## Observability
+
+When cluster virtualization is enabled, cluster logs and metrics are scoped to a virtual cluster or to the system interface.
```

Comment:
We should probably say a bit more here, namely, that application level metrics are visible from the system interface, but not vice versa.  Additionally, SQL Activity and Jobs information is scoped to the tenant where the activity occurs.  More generally, if you want to see SQL Activity and Jobs information, you should look at them from the application VC.